tests/conftest.py

```diff
@@ -1349,6 +1349,16 @@ def balsamic_customer(collaboration_id: str, customer_id: str) -> Customer:
     )
 
 
+@pytest.fixture
+def customer(collaboration_id: str, customer_id: str) -> Customer:
```

Comment:
It seems I wasn't inspired when naming the other fixtures (`mip_dna_customer` and `balsamic_customer`) 😓

Let's give this fixture at least a more descriptive name

```suggestion
def raredisease_loqusdb_customer(collaboration_id: str, customer_id: str) -> Customer:
```